tokenclient/tokenFlows.go

```diff
@@ -68,6 +85,7 @@ func NewTokenFlows(identity env.Identity, options Options) (*TokenFlows, error)
 		identity: identity,
 		tokenURI: identity.GetURL() + tokenEndpoint,
 		options:  options,
+		cache:    cache.New(15*time.Minute, 0), //nolint:gomnd
```

Comment:
What is the reason that the cache is never cleared?

Reply:
good point: expires after 15 minutes, cleanup using janitor every 10 minutes